app/helpers/ems_middleware_helper/textual_summary.rb

```diff
@@ -11,7 +11,7 @@ def textual_group_properties
   def textual_group_relationships
     # Order of items should be from parent to child
     items = []
-    items.concat(%i(middleware_servers middleware_deployments middleware_datasources))
+    items.concat(%i(middleware_servers middleware_deployments middleware_datasources middleware_messagings))
```

Comment:
Shouldn't this follow the other textual_summary ones and just be 
`%i(middleware_servers... middleware_messagins)` without the `items` array?
I know that this like this for a long time.